kubernetes/windows/main.ps1

```diff
@@ -39,6 +39,50 @@ function Start-FileSystemWatcher {
     Start-Process powershell -NoNewWindow .\filesystemwatcher.ps1
 }
 
```

Comment:
Do you have perf numbers for container log scale with & without AMA?

Reply:
I still have to organise these numbers. I was thinking will add that with the final merge and will send over the excel before I leave.